ipyvolume/volume.py

```diff
@@ -27,6 +27,7 @@ class Scatter(widgets.DOMWidget):
     vy = Array(default_value=None,allow_none=True).tag(sync=True, **array_serialization)
     vz = Array(default_value=None,allow_none=True).tag(sync=True, **array_serialization)
     selected = Array(default_value=None,allow_none=True).tag(sync=True, **array_serialization)
+    index = Integer(default_value=0).tag(sync=True)
```

Comment:
Lets call it sequence_index, since there may be more indices coming, hover_index, pick_index ...